src/OpenTracing/GlobalTracer.php

```diff
@@ -18,7 +18,7 @@
      *
      * @param Tracer $tracer
      */
-    public static function setGlobalTracer(Tracer $tracer)
+    public static function withGlobalTracer(Tracer $tracer)
```

Comment:
My bad, I did not read this. Do you believe a `set*` would be a better option? I honestly want to avoid them but I am also want to be idiomatic.

```
This method MUST be implemented in such a way as to retain the
immutability of the message, and MUST return an instance that has the
new and/or updated header and value.
```